cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -72,7 +72,16 @@ def created(self, key, value):
                 }
             )
             self["created_by"] = _created_by
-            date = clean_val("w", value, int, regex_format=r"\d{6}$")
+
+            date = 99999
```

Comment:
date `99999`? What kind of date is that???? :)

Reply:
:) this is a dirty hack, I didn't have a better idea there. The problem is that in this field we have date in format `YYYYWW` where `WW` is week number, and there might be two subfields `w`. We are supposed to choose the oldest date, so the initial 999999 is for comparing reasons. I thought it is not worth it to spend time for more sophisticated solutions.

Reply:
Ok, makes sense, I would just add a comment (even another PR) to explain this... if you read this code in one year I would wonder what happened here... :)